src/argilla/client/feedback/dataset.py

```diff
@@ -568,9 +569,7 @@ def delete_and_raise_exception(dataset_id: str, exception: Exception) -> None:
             for question in self.questions:
                 try:
                     datasets_api_v1.add_question(
-                        client=httpx_client,
-                        id=argilla_id,
-                        question=question.dict(include={"name", "title", "description", "required", "settings"}),
+                        client=httpx_client, id=argilla_id, question=json.loads(question.json())
```

Comment:
We shouldn't include extra fields, that why there was an `include=set()` before, IMO we should keep it

Reply:
Yes, but currently the API will discard them.

Ideally, we should have an SDK schema for each operation and create them from our feedback.schema instances:

```python
class QuestionCreate(BaseModel):
    title: str
    name: str
    settings: dict


api.add_question(client, QuestionCreate.from_raw(feedback_question)
```